sessions/js-array-methods-2/employees/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Array Methods 2: Employees</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <script src="./js/index.js" type="module"></script>
+  </head>
+  <body>
+    <h1>Human Ressources</h1>
+    <hr />
+    <div>
+      <form data-js="form1">
+        <label for="exercise1">Find by last name:</label>
+        <input type="text" id="exercise1" name="exercise1" />
+        <button>🔍 Search</button>
+      </form>
+      <div data-js="exercise1"></div>
```

Comment:
If you want to be very picky, you can use the [`<output>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output) element ^^